resqpy/olio/wellspec_keywords.py

```diff
@@ -272,41 +292,41 @@ def load_wellspecs(wellspec_file, well = None, column_list = []):
                 kf.skip_comments(fp)
                 if kf.blank_line(fp):
                     break  # unclear from Nexus doc what marks end of table
-                if kf.specific_keyword_next(fp, 'WELLSPEC') or kf.specific_keyword_next(fp, 'WELLMOD'):
+                if kf.specific_keyword_next(fp, "WELLSPEC") or kf.specific_keyword_next(fp, "WELLMOD"):
                     break
                 line = kf.strip_trailing_comment(fp.readline())
                 words = line.split()
-                assert len(words) >= len(columns_present),  \
-                    f'insufficient data in line of wellspec table {well} [{line}]'
+                assert len(words) >= len(
```

Comment:
I prefer the explicit line continuation character as this yapf line break is in a poor place

Reply:
Yeah me too, I missed that one